src/main/java/views/AcesUp/AcesUp.flt.html

```diff
@@ -188,22 +164,63 @@
         }
         option = this.value;
     };
+}
+function display(game) {
+console.log(game);
 
+$( '.columnOfCards .cardLocation' ).html("");
```

Comment:
After looking at this I see the reason you changed the display function. Good find! I messed around with the changes you made and it makes sense that we needed to reference the correct suit type!